README.md

```diff
@@ -1 +1,68 @@
-Please refer to the official documentation [here](https://ob.docs.wso2.com/en/latest/)
+<!--
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~ WSO2 LLC. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+
+# WSO2 Financial Services Accelerator
```

Comment:
Should be Open Banking Accelerator

Reply:
Fixed with https://github.com/wso2/financial-services-accelerator/pull/221/commits/e755ebde386e6560d9acf08b41c55992ca7d6562